Composer/packages/client/src/pages/setting/app-settings/AppSettings.tsx

```diff
@@ -127,7 +132,31 @@ const AppSettings: React.FC<RouteComponentProps> = () => {
           onToggle={onCodeEditorChange('wordWrap')}
         />
       </section>
-
+      <section css={section}>
+        <h2>{formatMessage('Application Language')}</h2>
+        <SettingDropdown
+          description={formatMessage('This is the language used for Composer’s user interface.')}
+          image={images.language}
+          options={[
+            {
+              key: 'en-US',
+              text: formatMessage('English (US)'),
+            },
+            // uncomment the following to test out the pseudo-localization:
```

Comment:
Do u think we can have a process.env that enables us to test out the localization feature?

Reply:
Good idea. I'll make that change.